src/cryptojwt/jwt.py

```diff
@@ -27,7 +28,9 @@ def utc_time_sans_frac():
 
     :return: A number of seconds
     """
-    return int(datetime.utcnow().timestamp())
+
+    now_timestampt = int(datetime.now(timezone.utc).timestamp())
```

Comment:
why not use `int(time.time())` ?

Reply:
Ah, long discussion. This was eventually what the powers that be agreed on.
I believe time.time() returns UTC but not absolutely sure.

Reply:
Ah, it seems platform specific what time.time() returns.

Reply:
Ah, I wonder if anyone would dare using something other than UTC, but YMMV. All good.